woocommerce/woocommerce-ios
# Adding displaying error with retry on sync error for bulk update screen

Part of: #6239 

### Description
This PR adds handling for errors during synching the product variations in the bulk update screen. If an error occurs an error notice is displayed with a retry button. The retry action re-tries the synch. 

### Testing instructions
1. Go to the products tab
2. Select a product with variations
3. Go to the list of variations
4. Turn the network off
5. Tap on the more button and select the "bulk update" option
6. You will see an error because the list will fail to load
7. Re-enable the network
8. Tap retry the list should load

### Screenshots


https://user-images.githubusercontent.com/96764631/156820809-c80a9b56-0304-4c4a-88a2-796848e56a75.mp4


---
